pkg/migration/runner.go

```diff
@@ -545,7 +545,7 @@ func (r *Runner) createNewTable(ctx context.Context) error {
 // alterNewTable applies the ALTER to the new table.
 // It has been pre-checked it is not a rename, or modifying the PRIMARY KEY.
 func (r *Runner) alterNewTable(ctx context.Context) error {
-	if err := dbconn.Exec(ctx, r.db, "ALTER TABLE %n.%n "+r.migration.Alter,
+	if err := dbconn.Exec(ctx, r.db, "ALTER TABLE %n.%n "+utils.TrimAlter(r.migration.Alter)+", ALGORITHM=COPY",
```

Comment:
Nice one without the need for temp table and I assume this is reasonably fast/ same speed as instant for empty table.

Reply:
Yeah, it's basically the same cost because it's altering an empty table at this point.